content-security-policy/support/echo-policy-multiple.py

```diff
@@ -0,0 +1,38 @@
+def main(request, response):
+    headers = [("Content-Type", "text/html")]
+    if "policy" in request.GET:
+        headers.append(("Content-Security-Policy", request.GET["policy"].replace("8", ";")))
```

Comment:
@mikewest  
It seems like GET parameters are split on a semicolon so it was not getting a full policy. This seems like a bad solution but I just used `8` as a special character and replaced it with a semicolon.